### PR TITLE
separate results collection from posting

### PR DESCRIPTION
**Pull request by k0s**

### CODE REVIEW

File: dzclient/client.py [hunk 1]
```diff
@@ -106,6 +101,40 @@ def datasets(self):
...
+    Note that the revision id can be 16 characters, maximum.
+
+    """
+
+    @classmethod
+    def create(cls, host, project, oauth_key, oauth_secret, collection):
+        """create a DatazillaRequest instance from a results collection"""
+
+        # get attributes from the collection
+        attributes = inspect.getargspec(foo).args[1:]
```
Comment (malini) [inline dzclient/client.py]: foo function isn't defined

File: dzclient/client.py [hunk 2]
```diff
@@ -41,22 +42,16 @@ def join_results(self, results):
             for test_name, values in tests.items():
                 suite.setdefault(test_name, []).extend(values)
 
+class DatazillaResultsCollection(object):
```
Comment (malini) [inline dzclient/client.py:45]: Can you add a docstring describing what this collection is?

Comment (k0s) [inline dzclient/client.py:45]: Will do

File: dzclient/client.py (hunk 1 above)
Comment (k0s) [inline dzclient/client.py]: How embarassing!  That's what i get for copy+pasting :(

**Pull request merged**